locations/spiders/deutschepost_de.py

```diff
@@ -0,0 +1,72 @@
+import scrapy
+import json
+import csv
+import re
+
+from locations.items import GeojsonPointItem
+from locations.hours import OpeningHours
+
+
+DAY_MAPPING = {1: 'Mo',
+               2: 'Tu',
+               3: 'We',
+               4: 'Th',
+               5: 'Fr',
+               6: 'Sa',
+               7: 'Su'}
+
+class DeutschepostDeSpider(scrapy.Spider):
+    name = "deutschepost_de"
+    allowed_domains = ["www.deutschepost.de"]
+
+    start_urls = []
+    with open('./locations/searchable_points/germany_centroids_80km_radius_country.csv') as csv_file:
```

Comment:
Can you use `start_requests()` instead of filling the `start_urls` list? For example:

https://github.com/alltheplaces/alltheplaces/blob/073f1338a5b495b2057b0ec19432502975f19758/locations/spiders/anytime_fitness.py#L13-L15

Reply:
Done.